src/Query/Mysql/AddTime.php

```diff
@@ -1,25 +1,26 @@
 <?php
 
+declare(strict_types=1);
+
 namespace DoctrineExtensions\Query\Mysql;
 
 use Doctrine\ORM\Query\AST\Functions\FunctionNode;
 use Doctrine\ORM\Query\Lexer;
+use Doctrine\ORM\Query\Parser;
+use Doctrine\ORM\Query\SqlWalker;
 
-/**
- * @author Pascal Wacker <hello@pascalwacker.ch>
- */
```

Comment:
Deleting the author tags everywhere should be discussed separately. Let's disable that rule.